components/writers/src/main/java/org/datacleaner/beans/writers/DeleteTableRowAnalyzer.java

```diff
@@ -0,0 +1,472 @@
+ * Free Software Foundation, Inc.
+ * 51 Franklin Street, Fifth Floor
+ * Boston, MA  02110-1301  USA
+ */
+package org.datacleaner.beans.writers;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Date;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicInteger;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+
+import org.apache.metamodel.BatchUpdateScript;
+import org.apache.metamodel.UpdateCallback;
+import org.apache.metamodel.UpdateScript;
+import org.apache.metamodel.UpdateableDataContext;
+import org.apache.metamodel.create.TableCreationBuilder;
+import org.apache.metamodel.csv.CsvDataContext;
+import org.apache.metamodel.delete.RowDeletionBuilder;
+import org.apache.metamodel.insert.RowInsertionBuilder;
+import org.apache.metamodel.query.FilterItem;
+import org.apache.metamodel.query.OperatorType;
+import org.apache.metamodel.query.SelectItem;
+import org.apache.metamodel.schema.Column;
+import org.apache.metamodel.schema.ColumnType;
+import org.apache.metamodel.schema.Schema;
+import org.apache.metamodel.schema.Table;
+import org.apache.metamodel.util.Action;
+import org.apache.metamodel.util.FileHelper;
+import org.apache.metamodel.util.Resource;
+import org.datacleaner.api.Analyzer;
+import org.datacleaner.api.Categorized;
+import org.datacleaner.api.ColumnProperty;
+import org.datacleaner.api.ComponentContext;
+import org.datacleaner.api.Concurrent;
+import org.datacleaner.api.Configured;
+import org.datacleaner.api.Description;
+import org.datacleaner.api.ExecutionLogMessage;
+import org.datacleaner.api.FileProperty;
+import org.datacleaner.api.FileProperty.FileAccessMode;
+import org.datacleaner.api.HasLabelAdvice;
+import org.datacleaner.api.Initialize;
+import org.datacleaner.api.InputColumn;
+import org.datacleaner.api.InputRow;
+import org.datacleaner.api.MappedProperty;
+import org.datacleaner.api.Provided;
+import org.datacleaner.api.SchemaProperty;
+import org.datacleaner.api.TableProperty;
+import org.datacleaner.api.Validate;
+import org.datacleaner.components.categories.WriteSuperCategory;
+import org.datacleaner.components.convert.ConvertToBooleanTransformer;
+import org.datacleaner.components.convert.ConvertToNumberTransformer;
+import org.datacleaner.connection.CsvDatastore;
+import org.datacleaner.connection.FileDatastore;
+import org.datacleaner.connection.SchemaNavigator;
+import org.datacleaner.connection.UpdateableDatastore;
+import org.datacleaner.connection.UpdateableDatastoreConnection;
+import org.datacleaner.data.MetaModelInputColumn;
+import org.datacleaner.descriptors.FilterDescriptor;
+import org.datacleaner.descriptors.TransformerDescriptor;
+import org.datacleaner.desktop.api.PrecedingComponentConsumer;
+import org.datacleaner.job.builder.AnalysisJobBuilder;
+import org.datacleaner.util.WriteBuffer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Named("Delete table row")
```

Comment:
Discussed the naming of this with @LosD because it sounds a bit funny to us. We suggest the name __Delete from table__ because it seems most consistent with the "Insert into table" and "Update table" components, which both lean on keywords from SQL. What do you think?